common/src/java/org/apache/hadoop/hive/conf/HiveConf.java

```diff
@@ -1663,7 +1663,8 @@ private static void populateLlapDaemonVarsSet(Set<String> llapDaemonVarsSetLocal
     HIVE_CBO_CNF_NODES_LIMIT("hive.cbo.cnf.maxnodes", -1, "When converting to conjunctive normal form (CNF), fail if" +
         "the expression exceeds this threshold; the threshold is expressed in terms of number of nodes (leaves and" +
         "interior nodes). -1 to not set up a threshold."),
-    HIVE_CBO_RETPATH_HIVEOP("hive.cbo.returnpath.hiveop", false, "Flag to control calcite plan to hive operator conversion"),
+    HIVE_CBO_RETPATH_HIVEOP("hive.cbo.returnpath.hiveop", "supported",
```

Comment:
You can provide a `StringSet` validator as parameter for the allowed values, e.g., check `HIVEFETCHTASKCONVERSION` below.

Reply:
fixed, thanks